sklearn/utils/fixes.py

```diff
@@ -26,6 +26,7 @@
 np_version = parse_version(np.__version__)
 sp_version = parse_version(scipy.__version__)
 sp_base_version = parse_version(sp_version.base_version)
+np_base_version = parse_version(np_version.base_version)
```

Comment:
You don't need this anymore, right?

Reply:
I would be in favor of keeping it for the sake of consistency. Otherwise we might start to add an remove it. And technically it could be considered public API so I would just add it once and for all.